src/command_modules/azure-cli-lab/azure/cli/command_modules/lab/validators.py

```diff
@@ -325,7 +312,18 @@ def _use_custom_image(namespace):
             raise CLIError(err.format(namespace.image, [x.name for x in custom_images]))
         else:
             namespace.custom_image_id = custom_images[0].id
-            namespace.os_type = custom_images[0].vhd.os_type
+
+            if custom_images[0].vm is not None:
```

Comment:
Can be simplified to 
`if custom_images[0].vm:`

The truthiness of `None` is false.